Sum of digits of a number.py

```diff
@@ -1,7 +1,13 @@
 q=0
-n=int(input("Enter Number: "))
-while(n>0):
- r=n%10
- q=q+r
- n=n//10
-print("Sum of digits is: "+str(q))
+while True:
```

Comment:
Something like this would be better.
```
q = 0
k = 3
for i in range(k):
    n = input(f"Please enter a number({k - i}): ")
    if n.isnumeric():
        n = int(n)
        break
    else:
        print("enter integer only: ")
        continue
  ```
  k naming is for OOP implementation and to avoid **magic number**.
  Using continue and break is good practice.
  
  You can opt for the ({k -i}) formatting.
  It is just there to show user how many tries are left.

Reply:
You can also use match case with "e", "exit", "exit()", "n", etc...for all neg values.
using isinstance of str checking, if yes then lower them, if that matches with these keywords then just break the loop.
Like this will enable exit the program.